exampleSite/content/posts/theme-documentation-basics.md

```diff
@@ -221,15 +221,6 @@ Text to display in the footer section
   footer = "Text in footer"
 ```
 
-### Displaying content on the homepage
-
-Content to display on homepage below the social icons, using the contents of `content/_index.md`.
-
```

Comment:
This should not be deleted, this documentation is still useful

Reply:
Yes, you're right. I only wanted to remove the reference to `enableFrontMatter`.

This should've been an atomic commit as well, oops. Re-added.